docs/documentation/pages/DECKHOUSE-FAQ.md

```diff
@@ -445,7 +445,20 @@ This feature is available in Enterprise Edition only.
 
    > Note that `--min-version` parameter will be ignored if you specify version above current rock-solid channel.
 
-   `dhctl mirror` supports digesting of the final set of Deckhouse images with the GOST R 34.11-2012 (Stribog) hash function, use the `--gost-digest` parameter to calculate it.
+   To pull Deckhouse images from a specific registry repository, specify that repository with the `--source` flag.
+   The optional `--source-login` and `--source-password` flags are used to authenticate to a given registry.
+   If they are omitted, mirroring will be performed anonymously.
+
+   For example, here is how you can pull Flant Edition images instead of the default Enterprise Edition ones:
```

Comment:
```suggestion
   An example of pulling Deckhouse Enterprise Edition images:
```

Reply:
This would be a bad example as pulling EE is the default behaviour and `--source` is not required to do it.